data/mods/ssb/moves.ts

```diff
@@ -907,6 +907,15 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 				}
 				return 5;
 			},
+			onModifyMove(move, source, target) {
+				if (!move.overrideOffensiveStat) {
+					move.overrideOffensiveStat = move.category === "Physical" ? 'atk' : 'spa';
+				}
+				if (!['atk', 'spa'].includes(move.overrideOffensiveStat)) return;
```

Comment:
I don't think this is necessary.

```suggestion
				if (move.overrideOffensiveStat && !['atk', 'spa'].includes(move.overrideOffensiveStat)) return;
```

This is enough, right?

Reply:
no because (I am assuming you meant to put a ! before the move.overrideOffensiveStat) it would make it so bane terrain doesn't work on moves that don't have their offensive stat overwritten.

That being said I can make the second if statement an else if as it cannot be true if the first is true, but it would make the code harder to read and wouldn't be run enough for it to be an issue. I can still make that chance if you think it is needed